CONTRIBUTING.MD

```diff
@@ -0,0 +1,64 @@
+Lyra Contributing Guidelines
+=======================
+
+New to Lyra? Check out "First PR" tag in the issues. These are issues have a lower level of effort for people looking to get their hands dirty in the Lyra codebase.
+
+## Guideline Contents
+
+There are a lot of ways to get involved and help out:
+
+- [Reporting an issue](#reporting-issues)
+- [Requesting features](#requesting-features)
+- [Submitting Pullrequests](#submitting-prs)
+- [Writing Documentation & tutorials](#documentation)
+- [Contributing Design Assets](#contributing-design)
+
+<a name="reporting-issues"></a>
+## Reporting Issues
+
+When creating a new issue for a bug please include the following:
+- **the browser you discovered the bug in**
+- **a description of the problem**
+- **a description of what you expected to happen**
+
+If it's applicable, include a gif of the interaction. You can capture your screen as a gif using [LiceCap](http://www.cockos.com/licecap/).*
+
+<a name="requesting-features"></a>
+## Requesting Features
+To request a new feature for the Lyra application, create an issue with the following:
+ - **What feature you'd like to see**
+ - **Why it is important to you**
+
+The more details you provide make it easier for people to scope the amount of work involved in creating the new feature.
+
+<a name="submitting-prs"></a>
+## How to submit a Pull Request
+We are trying to keep Lyra at full test coverage so any pull requests opened on the project must contain tests to be considered. If you have trouble writing tests, reach out to the [project maintainers](linktoprojectmaintainers) for help.
```

Comment:
Would it make sense to have a tag of some sort people can use to indicate it needs test coverage? Like [Needs-Tests] in the PR title?